internal/pkg/chainsync/internal/syncer/syncer_integration_test.go

```diff
@@ -120,193 +112,6 @@ func TestLoadFork(t *testing.T) {
 	assert.NoError(t, offlineSyncer.HandleNewTipSet(ctx, block.NewChainInfo("", "", left.Key(), heightFromTip(t, left)), false))
 }
 
-// Power table weight comparisons impact syncer's selection.
-// One fork has more blocks but less total power.
-// Verify that the heavier fork is the one with more power.
-// All blocks in this test follow protocol version 1 upgrade weighting rules.
-func TestSyncerWeighsPower(t *testing.T) {
```

Comment:
While modifying the state view this broke.  It would have been simple to fix but this test has been skipped for a long time and reflecting on its purpose leads me to believe that we are better deleting it.